openssl-builder/openssl.cnf

```diff
@@ -0,0 +1,350 @@
+#
```

Comment:
No need to closely review anything below - it's identical to what we were previously using when we built. It's only here temporarily until we get the version that SafeLogic generates.